chef_master/source/platforms.rst

```diff
@@ -35,38 +33,41 @@ The following table lists the Foundational platforms for the chef-client and oha
      - Version
    * - AIX
      - ``ppc64``
-     - ``6.1`` (TL6 or higher, recommended), ``7.1`` (TL0 SP3 or higher, recommended)
+     - ``6.1`` (TL6 or higher, recommended), ``7.1`` (TL0 SP3 or higher, recommended), ``7.2``
+   * - Amazon Linux
+     -
+     - current version
    * - CentOS
-     - ``i386``, ``x86_64``
+     - ``i386`` (5.x only), ``x86_64``
      - ``5``, ``6``, ``7``
    * - FreeBSD
      - ``i386``, ``amd64``
```

Comment:
As an aside, we only test on 10-i386, not 11-i386

Reply:
I assume we want to write it as FreeBSD `i386` is supported only on `10-STABLE` and for `11-STABLE` we would want to only support `amd64`?